basics/trey-hunner-zip-and-enumerate.md

```diff
@@ -153,28 +131,16 @@ Roll number of Michel is 28
 
 `enumerate()` is an amazing Built-in function offered by python. When used, gives us the index and the item combined. long story short! `enumerate` comes in handy while working with loops as it gives us access to both the item and it's index at the same time.
 
-### Syntax
-
-`enumerate(iterable, start=0)`
-
-### Parameters
-
-* Iterable: any iteration supporting object
-* Start: Starting index value, default is 0
-
-### Example 1: basic enumerate example
-
 ```python
 >>> even_nums = [2, 4, 6, 8, 10, 12]
-print(list(enumerate(even_nums)))
->>> 
+>>> print(list(enumerate(even_nums)))
 [(0, 2), (1, 4), (2, 6), (3, 8), (4, 10), (5, 12)]
 >>>
 ```
 
-Now before we use it in a loop let's take a look how we would do the same thing in a traditional way.
+Before we use it in a loop let's take a look how we would do the same thing in a traditional way.
```

Comment:
I think it would make sense to do this in the opposite order: first `enumerate()`, then `range(0, len(even_nums))`. Usually people copy the first code example they find, and we want people to use `enumerate()`.